Feature/doc 27

shikhanansi opened the pull request:
Hi Dylan,

To address this issue, I have done some major rework which includes:
Added section:
1. Quick Start to Tiny Cloud (This used to be this page: https://www.tiny.cloud/docs/get-started/first-steps/)
2. Changed names from:
a. Self-hosted Get Started to General Configuration Guide
b. Cloud Get Started to Cloud Deployment Guide
3. Removed the chapter Your first Steps and renumbered other chapters to accommodate this change.

Please review and let me know if these changes are in alignment with the expectations for [DOC-27 ](https://ephocks.atlassian.net/browse/DOC-27)

Kind Regards,
Shikha

r12s commented:
@shikhanansi I really like what you've done. I have feedback and I will note this in Jira.

shikhanansi commented:
Hi Dylan, 
I have made the following changes:
1. Changed the name to "Tiny Quick Start". 
2. Changed some text and moved around some content to provide getting started instructions for the user rather than giving them a choice between self-hosted and cloud.
3. Removed the references to the Getting Started guide from other places.
Please let me know if the changes look good enough to be merged.
Kind Regards,
Shikha

tinydylan commented on general-configuration-guide/basic-setup.md:
```diff
@@ -84,12 +84,12 @@ tinymce.init({
 ## Toolbar & menu configuration
 
-TinyMCE comes with a default set of toolbar controls out of the box, such as bold, italic, and text alignment buttons. Basically, the type of settings you would expect to find in any WYSIWYG editor.
+TinyMCE comes with a default set of toolbar controls out of the box, such as bold, italic, and text alignment buttons. Ideally, the type of settings you would expect to find in any WYSIWYG editor.
```
Do we even need the second sentence here?

shikhanansi replied:
I think the second sentence makes a point that tells the user what to expect. I'm keeping it for now.

tinydylan commented on general-configuration-guide/basic-setup.md:
```diff
@@ -143,17 +143,17 @@ tinymce.init({
 ### A quick look at menu and menubar controls
 
-Just as there is a `toolbar` option, there is also a `menu` option. In fact, there are two menu related options: `menu` and `menubar`. Let's take a quick look at the default menu items. You will see some of these settings used in the examples throughout the Get Started guide and all of them within the plugins and customization documentation.
+Just as there is a `toolbar` option, there is also a `menu` option. In fact, there is two menu related options: `menu` and `menubar`. Let's take a quick look at the default menu items. Some of these settings are used in the examples throughout this guide and all of them within the plugins and customization documentation.
```
I would have said "are"

shikhanansi replied:
The plural form of be 'are' does not seem to agree with the singular subject `two menu`.

tinydylan commented on general-configuration-guide/get-support.md:
```diff
@@ -1,11 +1,11 @@
 ---
 
-> If you landed here having worked through the [Get Started Guide]({{ site.baseurl }}/get-started/first-steps), congratulations, you are on you way to TinyMCE ninjahood :-) It's now time to deep-dive into [configuring the editor]({{ site.baseurl }}/configure/) and explor [plugin options]({{ site.baseurl }}/plugins/). We wish you well, and welcome you to TinyMCE. If you get stuck a great place to  ...
+> If you landed here having worked through the [General Configuration Guide]({{  site.baseurl }}/general-configuration-guide/advanced-install), congratulations, you are on you way to TinyMCE ninjahood :-) It's now time to deep-dive into [configuring the editor]({{ site.baseurl }}/configure/) and explore [plugin options]({{ site.baseurl }}/plugins/). We wish you well, and welcome you to TinyMCE. I ...
```
I'm surprised you didn't get rid of the "ninjahood" here :)

shikhanansi commented:
I quite like that reference myself :D

On Tue, Jul 31, 2018 at 3:45 PM, tinydylan <notifications@github.com> wrote:

> *@tinydylan* approved this pull request.
>
> I've left some minor comments, but otherwise this looks like a major
> improvement.
> ------------------------------
>
> In general-configuration-guide/basic-setup.md
> <https://github.com/tinymce/tinymce-docs/pull/723#discussion_r206400098>:
>
> >
>
>  ## Toolbar & menu configuration
>
> -TinyMCE comes with a default set of toolbar controls out of the box, such as bold, italic, and text alignment buttons. Basically, the type of settings you would expect to find in any WYSIWYG editor.
> +TinyMCE comes with a default set of toolbar controls out of the box, such as bold, italic, and text alignment buttons. Ideally, the type of settings you would expect to find in any WYSIWYG editor.
>
> Do we even need the second sentence here?
> ------------------------------
>
> In general-configuration-guide/basic-setup.md
> <https://github.com/tinymce/tinymce-docs/pull/723#discussion_r206400219>:
>
> > @@ -143,17 +143,17 @@ tinymce.init({
>
>  ### A quick look at menu and menubar controls
>
> -Just as there is a `toolbar` option, there is also a `menu` option. In fact, there are two menu related options: `menu` and `menubar`. Let's take a quick look at the default menu items. You will see some of these settings used in the examples throughout the Get Started guide and all of them within the plugins and customization documentation.
> +Just as there is a `toolbar` option, there is also a `menu` option. In fact, there is two menu related options: `menu` and `menubar`. Let's take a quick look at the default menu items. Some of these settings are used in the examples throughout this guide and all of them within the plugins and customization documentation.
>
> I would have said "are"
> ------------------------------
>
> In general-configuration-guide/get-support.md
> <https://github.com/tinymce/tinymce-docs/pull/723#discussion_r206400326>:
>
> >  description: Community and pro-grade support options.
>  keywords: forum forums url absolute relative security xss
>  ---
>
> -> If you landed here having worked through the [Get Started Guide]({{ site.baseurl }}/get-started/first-steps), congratulations, you are on you way to TinyMCE ninjahood :-) It's now time to deep-dive into [configuring the editor]({{ site.baseurl }}/configure/) and explor [plugin options]({{ site.baseurl }}/plugins/). We wish you well, and welcome you to TinyMCE. If you get stuck a great place to start is the [TinyMCE Forum](https://community.tinymce.com/).
> +> If you landed here having worked through the [General Configuration Guide]({{  site.baseurl }}/general-configuration-guide/advanced-install), congratulations, you are on you way to TinyMCE ninjahood :-) It's now time to deep-dive into [configuring the editor]({{ site.baseurl }}/configure/) and explore [plugin options]({{ site.baseurl }}/plugins/). We wish you well, and welcome you to TinyMCE. If you get stuck a great place to start is the [TinyMCE Forum](https://community.tinymce.com/).
>
> I'm surprised you didn't get rid of the "ninjahood" here :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tinymce/tinymce-docs/pull/723#pullrequestreview-141806554>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AmDEF1iK4uAvMCA8I7u_YymNE-Gt99SQks5uL-77gaJpZM4VOY0t>
> .
>
